test/merge/merge.bed

```diff
@@ -0,0 +1,4 @@
+ref1	1	4
+ref1	20	26
+ref1	32	45
+ref3	0	1
```

Comment:
Can you add the missing newline on the end please?  Also for the input SAM files.